src/test/java/co/org/osso/api/ApiApplicationTests.java

```diff
@@ -1,16 +1,40 @@
 package co.org.osso.api;
 
+import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import java.util.List;
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class ApiApplicationTests {
 
+	@Autowired
+	private BookController controller_target;
+
+	@Autowired
```

Comment:
I think at this point we don't *need* Spring to instantiate it for us. The service has no further dependencies on Spring components (I.e. it doesn't `@Autowire`) anything, so let's just test this the normal way with `private CustomerService target = new CustomerService();`. This should be in it's own test file, where we don't mention spring at all.

Reply:
I think I'm a little confused here, but done. I'll ask you my doubts my slack.